.rubocop.yml

```diff
@@ -17,3 +17,6 @@ Style/DocumentationMethod:
 Naming/MethodName:
   Exclude:
     - lib/f_service/base.rb
+
```

Comment:
Not on the scope of this PR, but what about dropping rubocop in favor of [standard](https://github.com/testdouble/standard)? We should stop fighting linters lol

Reply:
Nice to know, man! Thanks for letting me know about that gem.
I'll take a look...

And also, writing this PR, I  was thinking about to connect our f_rubocop here. :)

Reply:
I'd say we should not mix internal tools with open-source tools (I don't know if that's even possible). Anyway, it's a good opportunity to use standard on internal projects as well.

Reply:
Noted in: https://github.com/Fretadao/f_service/issues/31